sdk/communication/azure-communication-phonenumbers/CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 1.1.0-beta.1 (2022-01-24)
 
 ### Features Added
+- Support of DK and GB phone numbers acquisition.
```

Comment:
I would write this as: Support of Denmark (DK) and United Kingdom (GB) phone number acquisitions.

Reply:
This is the text Miguel suggested:

Users can now purchase United Kingdom (GB) toll free and geographic phone numbers for PSTN Calling
Users can now purchase Denmark (DK) toll free and geographic phone numbers for PSTN Calling